### PR TITLE
Add test coverage for get_task_name() in PR #4379

### PR DESCRIPTION
**Pull request by hclihn**
## Description
Add test coverage for get_task_name() in PR #4379

### CODE REVIEW

File: t/unit/tasks/test_trace.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from celery import group, uuid
 from celery import signals
 from celery import states
+from celery.app.task import Context
```
Comment (georgepsarakis) [inline t/unit/tasks/test_trace.py]: You should also import the function being tested, see also [here](https://travis-ci.org/celery/celery/jobs/304480942).

Comment (hclihn) [reply t/unit/tasks/test_trace.py]: The import is added.

**Comment by thedrow** [comment]
Thank you! Feel free to add yourself to the contributors file.

**Comment by thedrow** [comment]
Build failures on Windows are incorrect.

**Pull request merged**